### PR TITLE
Add median inference data metrics for workload

### LINKED ISSUE #1386: Add median inference data metrics for workload.
Current workload uses average inference data metrics

### PR DESCRIPTION
**Pull request by MarkGHX**
Fix #1386. Now the results contain both average inference data metrics and median inference data metrics in workload.

### CODE REVIEW

File: workload/resources/Workload.js [hunk 1]
```diff
@@ -677,7 +695,9 @@ class Workload {
       }
       logger.group('Result');
       const results = summarize(inferenceResults.inferenceTimeList);
-      logger.log(`Inference Time: <em style="color:green;font-weight:bolder;">${results.mean.toFixed(2)}+-${results.std.toFixed(2)}</em> [ms]`);
+      const results_median = summarizeMedian(inferenceResults.inferenceTimeList);
```
Comment (BruceDai) [inline workload/resources/Workload.js:698]: @MarkGHX Please replace the variable name ‘results_median’ with  'resultsMedian' of camelcase style, thanks.

**Review by BruceDai — COMMENTED** [review]
@MarkGHX The PR looks good, and I leave a comment about code style, please update it, thanks.

File: workload/resources/Workload.js (hunk 1 above)
Comment (MarkGHX) [reply workload/resources/Workload.js]: @BruceDai Thanks! This is fixed now.

**Comment by BruceDai** [comment]
Thanks @MarkGHX for enhance this feature.
@ibelem Please help merge this PR, thanks.

**Comment by ibelem** [comment]
Thanks @MarkGHX !

**Pull request merged**